src/main/ui/app/route/Show.jsx

```diff
@@ -39,61 +39,128 @@ export default class Show extends Component {
+
+</tr>);
             }
-            let statusClass = addon.status === "ACTIVE" ? "label label-default" :
-                              addon.status === "DEPRECATED" ? "label label-danger" : "label label-warning";
+          
             return (
-                    <div>
-                        <h1>
-                            <i className={icon} aria-hidden="true"></i>
-                            {addon.name}
-                            <span className={statusClass}>{addon.status}</span>
-                        </h1>
-                        <h3>{addon.description}</h3>
-                        {hosted}
-                        <p>
-                            Maintained by:
-                            {addon.maintainers.map(m => {
+
+                    <div className="showpage-body">
+<Panel header={showtitle}>
+<h4 className="lead">{addon.description}</h4>
+<div className="col-md-6 col-sm-12 col-xs-12 left-margin">
+<Table striped condensed hover>
+<tbody>
+<tr>
+<th>Type :</th>
+<td><h5>{addon.type}</h5></td>
+</tr>
+{check}
+<tr>
+        <th>Maintained by: </th>
+        <td><h5>{addon.maintainers.map(m => {
                                 if (m.url) {
                                     return (
-                                            <a target="_blank" href={m.url}>{m.name}</a>
-                                    )
+                                    
+                                   <Button bsSize="small" href={m.url}>{m.name}</Button>                                   )
                                 } else {
-                                    return <span>{m.name}</span>
+                                    return( 
+                                    <Button bsSize="small">{m.name}</Button>
+
+                                    )
                                 }
                             })}
-                        </p>
-                        Versions:
-                        <ul>
-                            {addon.versions.map(v => {
+    </h5></td>
+</tr>
+<tr>
+        <th>Version Count  <OverlayTrigger placement="right" overlay={versioncountmeaning}>
+      <Glyphicon glyph="glyphicon glyphicon-question-sign" />
+    </OverlayTrigger>: </th>
+	<td>{addon.versioncount}</td>
+</tr>
+<tr>
+<th>
+Latest Version:
+</th>
+<td>
+<Badge>{addon.latestversion}</Badge>
+</td>
+</tr>
+</tbody>
+</Table>
+</div>
+                     
+<Table condensed hover>
+    <thead>
+      <tr>
+        <th className="col-md-2 col-sm-2 col-xs-2">Version<OverlayTrigger placement="right" overlay={versionmeaning}>
+      <Glyphicon glyph="glyphicon glyphicon-question-sign" />
+    </OverlayTrigger></th>
+        <th className="col-md-6 col-sm-6 col-xs-6">OpenMRS Version Requirement<OverlayTrigger placement="right" overlay={requirementmeaning}>
+      <Glyphicon glyph="glyphicon glyphicon-question-sign" />
+    </OverlayTrigger></th>
+        <th className="col-md-2 col-sm-2 col-xs-2">Not sure</th>
```

Comment:
I wasn't able to figure out the info it provides.It's only output is 1,2,3 or something like that so shall I remove that column?